src/main/java/com/google/edith/GroceryDataReader.java

```diff
@@ -38,16 +38,16 @@ public DealItem readFile(String itemName) throws IOException {
     record = reader.readNext();
     record = reader.readNext();
 
-	while ((record = reader.readNext()) != null) {
+    while ((record = reader.readNext()) != null) {
       if (record[0].equals(itemName)) {
         List<DealItem> dealItems = new ArrayList<DealItem>();
 
         for (int i = 0; i < STORES.size(); i++) {
           DealItem item = new DealItem();
           item.setStore(STORES.get(i));
-          item.setPrice(record[i*3+1]);
-          item.setWeight(record[i*3+2]);
-          item.setComment(record[i*3+3]);
+          item.setPrice(record[i * 3 + 1]);
```

Comment:
it's 3 because of how the csv file is structured, each store has 3 columns of data so if the loop variable i corresponds to store, the individual cells will be i*3. i'll write a comment for that and store it in a variable